### ISSUE TITLE
Trying to make the core run

### ISSUE THREAD

**Issue by asturur** [spark2k06/PCXT_MiSTer#16]
Hello, i m trying to run this core with little to not success.

I copied the core in sdcard/_computer folder.

And i usb0/games/PCXT i have the following

<img width="145" alt="image" src="https://user-images.githubusercontent.com/1194048/176970300-ea84756c-401a-48de-b2d2-d210a6400783.png">

Where hdd.img has been created running this command here:
qemu-system-x86_64 -boot menu=on,strict=on -hda freedos.img -cdrom FD13LIVE.iso -device adlib -soundhw pcspk -usbdevice mouse
Using freedos 1.3 and then renaming freedos.img to hdd.img.
The partition is formatted with standard NON 32 fat.

Before running the core i run the uart script that return me the message:
`hdd.img has been mounted on COM1`

Then i start the core, i fot an adlib sound, a pc speaker beep, get the ram check, but none of the drives found and an error either A->A boot error or C->C boot error.

What could i be doing wrong?

**Comment by asturur** [comment]
As an added note, i played by chance with the video filter option, and now i can't get any display anymore

**Comment by spark2k06** [comment]
Do not use a video filter while this core is in beta, and this is the content of the **script**:

![image](https://user-images.githubusercontent.com/3509674/176987382-95b9b8ed-46a3-477f-a43b-3547d58abb57.png)

You should be aware that the use of this script to mount the image is temporary, until we get **VHD** loading by **OSD**. As you can see, the message that the image has been mounted on **COM1** is always displayed, whether the previous instruction failed or not.

In short, save everything to **sdcard/games/PCXT** instead of **usb0/games/PCXT**

**Comment by asturur** [comment]
Ok so i tried both moving the files in sdcard and then changing the script to work from my paths, i then decided to run the script manually, and the console output is that  `new is not a command`.

I can run `tmux -h` and i get a short option list, but writing the script command tells me that `new` is not a known command.

**Comment by asturur** [comment]
Oh also if i try to run serdrive alone ( thinking that is a binary? ) i get a `permission denied` i verify that it has the executable permission anyway and tried to switch it to chmod 777

**Comment by spark2k06** [comment]
I don't know why it doesn't work, that's how it should be displayed when **serdrive** is running correctly:

![image](https://user-images.githubusercontent.com/3509674/177001537-6f02065a-de22-4cda-bab4-6e7c6e0029ad.png)

**Comment by asturur** [comment]
does serdrive need anything to run outside itself?

**Comment by spark2k06** [comment]
You don't need anything else, just the **serdrive** application itself... you can see in this **screenshot** that it works perfectly, at least in my case. I don't understand the circumstances why it doesn't work for you, sorry.

**Comment by asturur** [comment]
ok somehow this executable can't be run from my particular installation. I started a fresh mister and it works from sdcard, serdrive complains about my 512meg drive and asks me to determine the geometry and i m failing at it.
Which is a hdd.img size that works out of the box?

**Comment by spark2k06** [comment]
In the corresponding thread on the misterfpga forum, the subject of geometry is discussed... I suggest you take a look at it, I'm closing the issue:

https://misterfpga.org/viewtopic.php?p=54481#p54481

**Comment by asturur** [comment]
i managed to get it recognized with a geometry of 1024:64:16, since with 63 wasn't really possible to find an integer the app liked.
Now the core sees it but it says, `partition signature != 55aa`, the disk is bootable from qemu anyway.
I ll try to recreate it again

**Comment by asturur** [comment]
ok ok i got it.
freedos booted and warned me i need a 386 :D
So i got with msdos 3.30 and that seems to work. A bit slow, as it was slow my old pc in listing content of dirs.

What is the best dos version you suggest for this kind of cpu? i do remember my pc coming with 3.3 or 3.2

**Comment by spark2k06** [comment]
Great, I use msdos 6.22 with no problems. I suggest you use Juko ST's BIOS instead of Sergey's, I've noticed that the screen scrolling in MSDos on this one is slow. However, I will tell you that it is a bit slower than an original 4.77Mhz, in the future it will be revised to make it cycle accurately.

**Comment by asturur** [comment]
the py script to transform the bios doens't work on my mac, and i don't know python enough to make it work.
Is the other bios shareable as it is?
i did not know dos 6.22 could run on the 8088, at least i get edit.
Gonna update it.

It feels indeed a little bit slow, but just a tad bit. Digger was anyway slow on my pc, i can't remember exactly how much compared to what i have seen now.

**Comment by asturur** [comment]
Also, dumb question.
My understandting is that the 8088 was at 4.77mhz because of the ISA bus ( a third of the bus clock).
Is the isa bus replicated in this implementation? Do you plan any kind of speed toggles? 4.77 vs more Mhz

**Comment by spark2k06** [comment]
On GitHub I only share Sergey's BIOS because it is Open Source, Juko ST's is not. Regarding the speed, the priority is 4.77Mhz of exact cycle, in the future, yes, I plan to provide some turbo mode.
